CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to the ODD SDK
+
+Thank you for helping to improve the project! We are so happy that you are contributing! 💖
+
+You can contribute to the ODD SDK at any level. Whether you are new to web development or have been at it for ages doesn't matter. We can use your help!
+
+**No contribution is too small, and all contributions are valued.**
+
+This guide will help you to get started. It includes many details, but don't let that turn you away. Consider this a map to help you navigate the process, and please reach out to us with any questions and concerns in the `#odd-sdk` channel on our [Discord server][discord].
+
+## Conduct
+
+Please review our [Code of Conduct][coc] which describes the behavior we expect from all contributors. Please be kind, inclusive, and considerate when interacting with contributors and maintainers.
+
+## Contributing in issues
+
+You can contribute by [opening an issue][issue] to request a feature or report a bug. We also welcome comments on an issue, bug reproductions, and pull requests if you see how to address an issue.
+
+If you are still determining the details of a feature request, you can start with a [discussion][discussions] where we can informally discuss what you have on your mind.
+
+**Anyone can participate in any stage of contribution**. We urge you to
+join in the discussion around bugs and comment on pull requests.
+
+### Asking for help
+
+If you have reviewed our [documentation][docs] and still have questions or are having problems, ask for help in the `#odd-sdk` channel on our [Discord server][discord] or [start a discussion][discussions].
+
+### Submitting a bug report
+
+We provide a template to give you a starting point when submitting a bug report. Please fill this template in with all relevant information, but feel free to delete any sections that do not apply.
+
+The most important information is describing the problem and how it impacts you. We also invite you to propose a solution, which could be a description of expected behavior.
+
+We very much appreciate it if you could include a [short, self-contained, correct example][sscce] that demonstrates the issue.
+
+## Pull requests
```

Comment:
I wonder if we should mention something about how contributors need to make sure to sign their commits?(assuming we still have that setup?)

Reply:
Yes! That's a great point, we should include that. I'll add something. 👌

Reply:
Alright, added this note:

https://github.com/oddsdk/ts-odd/blob/5c1664e1d8a2abc38eddab1b01cb77034ab07751/CONTRIBUTING.md?plain=1#L42

Reply:
nice, looks great!